modules/3d_image_transforms.ipynb

```diff
@@ -457,23 +410,23 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "image shape: (1, 334, 334, 55)\n",
-      "label shape: (1, 334, 334, 55)\n",
+      "image shape: (1, 512, 512, 55)\n",
+      "label shape: (1, 512, 512, 55)\n",
       "image affine after Spacing:\n",
-      "[[  0.          -1.5          0.           0.47680664]\n",
-      " [ -1.5          0.           0.           0.47680664]\n",
-      " [  0.           0.          -5.         270.        ]\n",
-      " [  0.           0.           0.           1.        ]]\n",
+      "[[ 0.0000000e+00 -9.7656202e-01  0.0000000e+00  4.7683716e-07]\n",
+      " [-9.7656202e-01  0.0000000e+00  0.0000000e+00  4.7683716e-07]\n",
+      " [ 0.0000000e+00  0.0000000e+00 -5.0000000e+00  2.7000000e+02]\n",
+      " [ 0.0000000e+00  0.0000000e+00  0.0000000e+00  1.0000000e+00]]\n",
       "label affine after Spacing:\n",
-      "[[  0.          -1.5          0.           0.47680664]\n",
-      " [ -1.5          0.           0.           0.47680664]\n",
-      " [  0.           0.          -5.         270.        ]\n",
-      " [  0.           0.           0.           1.        ]]\n"
+      "[[ 0.0000000e+00 -9.7656202e-01  0.0000000e+00  4.7683716e-07]\n",
+      " [-9.7656202e-01  0.0000000e+00  0.0000000e+00  4.7683716e-07]\n",
+      " [ 0.0000000e+00  0.0000000e+00 -5.0000000e+00  2.7000000e+02]\n",
+      " [ 0.0000000e+00  0.0000000e+00  0.0000000e+00  1.0000000e+00]]\n"
      ]
     }
    ],
    "source": [
-    "data_dict = orientation(data_dict)\n",
+    "data_dict = orientation(datac_dict)\n",
```

Comment:
Is this a typo?

Thanks.

Reply:
it's not a typo, previously I swap the cells but forgot to change the variables. now it's tested with v0.8.1. I think we are good for tagging for tutorial 0.8.1 after this PR? do you have any concerns?